src/app/docs/help-inline/help-inline-docs.component.html

```diff
@@ -7,17 +7,23 @@
     The help inline component creates a small button next to a field. When clicked they can trigger an action that displays more information about that field that the user might need information on.
   </sky-docs-demo-page-summary>
 
-  <!-- <sky-docs-demo>
+  <sky-docs-demo>
+    <sky-help-inline>
+    </sky-help-inline>
```

Comment:
Should this do something when clicked?

Reply:
Unsure. You want it to show an `alert()`?

Reply:
Added an `alert`.

Reply:
Yeah. I think your code example already does that. Lets just make them match.